src/gui/Brickwall/BrickwallPlot.cpp

```diff
@@ -98,7 +98,7 @@ BrickwallPlot::BrickwallPlot (State& pluginState, dsp::brickwall::Params& brickw
                                 .sampleRate = sampleRate,
                                 .fftOrder = fftOrder,
                             }),
-      brickwall (brickwallParams),
+      brickwall (brickwallParams, *pluginState.nonParams.brickwallExtraState),
```

Comment:
For some reason GitHub won't let me comment on the actual line, but further down in this file, we're painting the full background of the Brickwall plot. I think that's probably the root cause of why the spectrum isn't showing when we send it `toBack()`.

I guess we need to set it up so that the editor component paints the background instead?